### ISSUE TITLE
Optimize rendering 

### ISSUE THREAD

**Issue by vicentecarro** [morevnaproject/remake#6]
I have a 22' project. Remake is working properly, but if I change or add 1 only frame of the project then Remake will render it full (the 22', that's more than 38000 frames...).
It would be great if Remake could detect the internal changes of the blender file and only render the changed part, reusing the previously rendered frames of the last Remake run.
Not sure if this can be done, but just an idea... Currently it's a pain using Remake with a full length project.

**Comment by morevnaproject** [comment]
Yes, the delta-rendering can dramatically increase rendering speed for many cases. But considering the complexity of the problem and high variety of file formats used, my opinion is that delta-rendering should be done by the rendering application, not Remake. I think the best way is to ask blender developers to implement that. ^_^

But at current state of things there are two workarounds possible:
1. Split your project into pieces. Each piece should be separate "Remake project". You can render each piece when you working on the project and stitch them all together at the final stage.
2. Use new "deps" command introduced in the fbf6c1af749a82435b658fdc759027f6a5ec0b76. Run "../remake project.blend deps", then open project.blend and watch your changes directly in the sequence file. See http://github.com/morevnaproject/remake/wiki/Man-page for details.

**Comment by morevnaproject** [comment]
Starting from d234a53ec Remake is now supports dependency caching. That dramatically increases speed of dependency detection.

**Comment by vicentecarro** [comment]
Thanks. I'll try this new feature as soon as possible.
